src/libraries/System.Linq.Queryable/tests/OrderDescendingTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public sealed class OrderDescendingTests : EnumerableBasedTests
+    {
+        [Fact]
+        public void OrderDescending1()
+        {
+            var count = (new int[] { 0, 1, 2 }).AsQueryable().OrderDescending().Count();
+            Assert.Equal(3, count);
```

Comment:
We should validate that the output is sorted as expected.

Reply:
@deeprobin could you take another look at the test feedback. I don't believe it's been resolved.

Reply:
Please take a look at the corresponding OrderByDescendingTests.cs:

https://github.com/dotnet/runtime/blob/6a21dbf9ffa5d9fb0adb931b4fad97256e0565ee/src/libraries/System.Linq.Queryable/tests/OrderByDescendingTests.cs

There's a bunch of tests missing, including basic smoke testing and null argument validation testing.

Reply:
Oh I confused the tests with the non-query tests. Thanks for bringing it to my attention :D.